### PR TITLE
Update an-introduction-to-iota.md

### PR DESCRIPTION
**Pull request by Phyloiota**
created draft content

# Description of change

Please write a summary of your changes and why you made them. 

## Links to any relevant issues

Be sure to reference any related issues by adding `fixes issue #`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)
- Enhancement (a non-breaking change which adds functionality)
- Documentation Fix
- Documentation Enhancment

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

**Comment by Phyloiota** [comment]
as a starting point okay, can be extended from here on

File: docs/learn/about-iota/an-introduction-to-iota.md [hunk 1]
```diff
@@ -5,30 +5,70 @@ description: A basic introduction to the IOTA DLT Technology. The use-cases and
 ---
 
 
-# Heading 1 (Page Title)
+# Introduction to IOTA
 
-content - introduction to the topic
+**This topic describes some of the main differences between the Tangle and a blockchain.**
```
Comment (Dr-Electron) [inline docs/learn/about-iota/an-introduction-to-iota.md:10]:
```suggestion
**This topic describes some main differences between the Tangle and a blockchain.**
```
Sound nicer to me :)

File: docs/learn/about-iota/an-introduction-to-iota.md [hunk 2]
```diff
@@ -5,30 +5,70 @@ description: A basic introduction to the IOTA DLT Technology. The use-cases and
...
+
+
+## **Consensus in a blockchain**
+
+In blockchains, the network is split into miners and users. Miners consume large amounts of computing power to complete the proof of work (PoW) required to chain the blocks together.
+Miners are incentivized to validate messages and do PoW because of the following:
+The fees that users are willing to pay to have their messages included in a block
+The reward that the network gives them for mining the block before other miners
+The only way to reverse messages in a blockchain is to mine a new blockchain in the time it takes other miners to mine a single block. To do so, a miner would need 51% of the network's ability to do PoW known as hash power.
+As a result, requiring PoW secures blockchain networks by making it difficult to attack, change, or stop. The more miners that mine, the more the secure the network.
```
Comment (Dr-Electron) [inline docs/learn/about-iota/an-introduction-to-iota.md:51]:
```suggestion
As a result, requiring PoW secures blockchain networks by making it difficult to attack, change, or stop. The more miners participate, the more secure the network is.
```

Sounded a little weird to me. Not sure if m suggestion is better 😅

File: docs/learn/about-iota/an-introduction-to-iota.md [hunk 3]
```diff
@@ -5,30 +5,70 @@ description: A basic introduction to the IOTA DLT Technology. The use-cases and
...
+
+## **Consensus in the Tangle**
+
+In the Tangle, messages require no fees because the network has no miners.
+In the Tangle, PoW is not used to secure the network. Instead, PoW is used only to discourage spam messages.
+To reach a consensus, all IOTA nodes validate messages and use different functions alongside messages in their confirmation.
+Currently messages will only considered valid if they have a refference to a milestone. These milestones are issued by a special network node - the Coordinator.
```
Comment (Dr-Electron) [inline docs/learn/about-iota/an-introduction-to-iota.md:58]:
```suggestion
Currently messages will only be considered valid if they have a reference to a milestone. These milestones are issued by a special network node - the Coordinator.
```

File: docs/learn/about-iota/an-introduction-to-iota.md [hunk 4]
```diff
@@ -5,30 +5,70 @@ description: A basic introduction to the IOTA DLT Technology. The use-cases and
...
+
+## **The coordinator**
+
+**The Coordinator is a client that sends signed messages called milestones that nodes trust and use to confirm messages. This topic describes how nodes use milestones to determine which messages are confirmed.**
+
+ - The Coordinator is temporary. We will transition into removing the Coordinator:Coordicide in the upcomming IOTA 2.0 Metwork. Below is how the Coordinator is currently employed within IOTA.
```
Comment (Dr-Electron) [inline docs/learn/about-iota/an-introduction-to-iota.md:64]:
```suggestion
 - The Coordinator is temporary. We will transition into removing the Coordinator: Coordicide in the upcoming IOTA 2.0 network. Below is how the Coordinator is currently employed within IOTA.
```

File: docs/learn/about-iota/an-introduction-to-iota.md [hunk 5]
```diff
@@ -5,30 +5,70 @@ description: A basic introduction to the IOTA DLT Technology. The use-cases and
...
+
+Messages in the Tangle are considered for confirmation only when they are directly or indirectly referenced by a milestone that has been validated by nodes.
+
+To allow them to recognize milestones, all nodes in the same IOTA network are configured with the Merkle root address of a Coordinator that they trust to confirm messages. Using this address, nodes can validate the signatures in milestones to verify whether they are signed by their trusted Coordinator.
+
+To make sure that new messages always have a chance of being confirmed, the Coordinator sends indexed milestones at regular intervals. This way, nodes can compare the indexes of their milestones to check whether they are synchonized with the rest of the network.
```
Comment (Dr-Electron) [inline docs/learn/about-iota/an-introduction-to-iota.md:70]:
```suggestion
To make sure that new messages always have a chance of being confirmed, the Coordinator sends indexed milestones at regular intervals. This way, nodes can compare the indexes of their milestones to check whether they are synchronized with the rest of the network.
```

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

**Review by Dr-Electron — APPROVED** [review]

**Pull request merged**